### ISSUE TITLE
It would be practical if there was an option/feature to paste the character instead of copying it.

### ISSUE THREAD

**Issue by mzso** [phts/firefox-special-characters#5]
Hi!

I think pasting the character would be more practical most of the time, unless I want to use the character mutliple times. 
So an option for it would be great. Or maybe a feature to paste with middle or right click, and give back focus to the page.

**Comment by phts** [comment]
@mzso yes. I also was thinking about pasting. I will add it to my todo list.

Thank for the request.

**Comment by mzso** [comment]
> 
> 
> @mzso yes. I also was thinking about pasting. I will add it to my todo list.
> 
> Thank for the request.

Thanks!
